cryptography/kzg_multi_open/src/fk20/cosets.rs

```diff
@@ -43,7 +43,9 @@ pub fn coset_gens(num_points: usize, num_cosets: usize, bit_reversed: bool) -> V
     use bls12_381::ff::Field;
 
     // Compute the generator for the group containing all of the points.
-    // TODO: generating the whole group, just to get the generator is inefficient
+    //
+    // Note: generating the whole group, just to get the generator is inefficient
+    // However, this code is not on the hot path, so we don't optimize it.
```

Comment:
This is only called upon startup, we could just compute the generator given we know the largest primitive root of unity. I would only do this if we benchmark it to be an issue